Dockerfile

```diff
@@ -16,11 +16,11 @@ RUN npm rebuild loady
 RUN npm rebuild node-jq
 RUN npm rebuild sqlite3
 
-FROM node:16-alpine as app
+FROM node:20 as app
```

Comment:
This makes the image quite a bit larger but I was wasting time trying to figure out alpine issues so 🤷🏼

Long-term we'll be working in the new tests repo anyway.